test/RegisterDomains.ts

```diff
@@ -0,0 +1,94 @@
+const { expect } = require("chai");
+const {
+    loadFixture,
+} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
+const { ethers } = require("hardhat");
+
+const undefinedAddress = "0x0000000000000000000000000000000000000000";
+
+describe("RegisterDomains contract", function () {
+    async function deployTokenFixture() {
+        const [owner, addr1] = await ethers.getSigners();
+        const hardhatToken = await ethers.deployContract("RegisterDomains");
```

Comment:
Can this const have a better name? -1

Reply:
renamed and moved setup state to `beforeAll`